certification/internal/shell/validate_operator_bundle.go

```diff
@@ -0,0 +1,54 @@
+package shell
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/komish/preflight/certification"
+	"github.com/sirupsen/logrus"
+)
+
+type ValidateOperatorBundlePolicy struct {
+}
+
+func (p ValidateOperatorBundlePolicy) Validate(bundle string, logger *logrus.Logger) (bool, error) {
+	command, err := exec.Command("which", "operator-sdk").CombinedOutput()
+	if err != nil {
+		logger.Error("Error finding the operator-sdk command: ", err)
+		return false, nil
```

Comment:
Thank you Emily for the great work! I took a closer look, and I suggest to return `false, err` because this is an erroneous condition. I see a bit of inconsistency in other checks as well, and I am going to fix them. Any other opinion on this?

Reply:
Yeah I think that you are right I think that in this case it probably should have returned the error. I am going to remove the which command and just use the operator-sdk instead of using the path name so I am just getting rid of this.